pkg/cmd/codespace/ssh.go

```diff
@@ -79,15 +79,14 @@ func newSSHCmd(app *App) *cobra.Command {
 			install it in your %[1]sDockerfile%[1]s or, for codespaces that use Debian-based images,
 			you can add the following to your %[1]sdevcontainer.json%[1]s:
 
-				"features": {
-					"ghcr.io/devcontainers/features/sshd:1": {
-						"version": "latest"
-					}
-				}
+			    "features": {
+			        "ghcr.io/devcontainers/features/sshd:1": {
+			             "version": "latest"
+			        }
+			    }
```

Comment:
I don't know if these space indentations will be preserved when we pull this up as I think go formatter prefers tabs.